misc/scripts/vfile-swarm-setup.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#source ../misc/scripts/commands.sh
+source commands.sh
+filename=$1
+
+i=0
+idx=0
+while IFS= read -r line || [ -n "$line" ]
+do
+    if [ $i == 0 ]
+    then
+        # NODE_COUNT is the total number of nodes that in the swarm cluster
+        NODE_COUNT=$line
+        echo "NODE_COUNT  $NODE_COUNT"
+    fi
+    if [ $i == 1 ]
+    then
+        # MGR_COUNT is the total number of manager nodes in the swarm cluster
+        MGR_COUNT=$line
+        echo "MGR_COUNT $MGR_COUNT"
```

Comment:
don't we need to fail the script if the MGR_COUNT is missing? inshort what happens for the following vfile-swarm-config.txt (when one of the value from NODE_COUNT or MGR_COUNT is missing)

```
3
10.161.101.247
10.161.98.108
10.161.98.109
```

Reply:
Done.